README.md

```diff
@@ -124,6 +124,7 @@ flutter pub run grinder update-code-segments
     * Create the app bundle with `flutter build appbundle`.
     * Upload to the Play store console.
     * Publish the Play store release.
+    * Upload to Google play store [instructions](https://docs.google.com/document/d/1M3NUVdVo0jHXkyTzZ8d-YRDA6gCqOl0JWV8enbwJF2w/edit?usp=sharing).
```

Comment:
I think these steps are covered in the google doc, so I think step 4 can just be shortened to include a link to the doc  with no steps.